vignettes/contingency-tables-vignette.Rmd

```diff
@@ -186,19 +261,22 @@ When can use this new design on the data obtained for the unrestricted balanced
 ```{r}
 safe.prop.test(ya = ya, yb = yb, designObj = differenceRestrictedSafeDesign)
 ```
-We see that we now have obtained a much higher e-value, i.e., more evidence for the alternative hypothesis. We can also see in the design that if the true difference between the groups would be 0.3, we would only need to collect 58 data blocks instead of 95. However, one needs to be careful: if the true difference would be smaller than 0.3, we could miss evidence and would need to continue experiments longer than with the unrestricted safe tests: a good illustration can be found in (Turner, 2021)[#TurnerRef], figure 5.
+We see that we now have obtained a much higher e-value, i.e., more evidence for the alternative hypothesis. We can also see in the design that if the true difference between the groups would be 0.3, we would only need to collect 54 data blocks instead of 95. However, one needs to be careful: if the true difference would be smaller than 0.3, we could miss evidence and would need to continue experiments longer than with the unrestricted safe tests: a good illustration can be found in [Turner et al., 2021](#TurnerRef), figure 4.
```

Comment:
You can make the number 54 blocks dynamic by replacing it by 

`differenceRestrictedSafeDesign$nPlan[3]`

I didn't set a seed so I got 58 after running the commented out code on L238.

Reply:
We see that we now have obtained a much higher e-value, i.e., more evidence for the alternative hypothesis. We can also see in the design that if the true difference between the groups would be 0.3, we would only need to collect `differenceRestrictedSafeDesign$nPlan[3]` data blocks instead of 95. However, one needs to be careful: if the true difference would be smaller than 0.3, we could miss evidence and would need to continue experiments longer than with the unrestricted safe tests: a good illustration can be found in [Turner et al., 2021](#TurnerRef), figure 4.

Reply:
> You can make the number 54 blocks dynamic by replacing it by
> 
> `differenceRestrictedSafeDesign$nPlan[3]`
> 
> I didn't set a seed so I got 58 after running the commented out code on L238.

unfortunately that does not work :( Not surprised you get a slightly different answer with a different seed, that's okay